sydent/http/servlets/threepidunbindservlet.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2014 OpenMarket Ltd
+# Copyright 2018 New Vector Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+from sydent.http.servlets import get_args, jsonwrap
+from sydent.hs_federation.verifier import NoAuthenticationError
+from signedjson.sign import SignatureVerifyException
+
+from twisted.web.resource import Resource
+from twisted.web import server
+from twisted.internet import defer
+
+class ThreePidUnbindServlet(Resource):
+    def __init__(self, sydent):
+        self.sydent = sydent
+
+    def render_POST(self, request):
+        self._async_render_POST(request)
+        return server.NOT_DONE_YET
+
+    @defer.inlineCallbacks
+    def _async_render_POST(self, request):
```

Comment:
does this not need a generic `Exception` handler?

Reply:
yeah, true